requirements/pytorch/devel.txt

```diff
@@ -4,11 +4,5 @@
 # install all extra dependencies for full package testing
 -r ./extra.txt
 
-# install all loggers for full package testing
--r ./loggers.txt
-
 # extended list of dependencies for development and run lint and tests
 -r ./test.txt
-
-# install all extra dependencies for running examples
--r ./examples.txt
```

Comment:
But `devel.txt` is meant for developers, not for CI; that we import it in CI is just our simplification, and no developer shall have limited experience just because of convenience... :)

Reply:
Our examples use many different requiremenets that are not included in this file (nvidia dali for example). Developers do not need to install all of them to develop PL, especially if they are untested.